src/work-allocation-2/containers/task-list/task-list.component.html

```diff
@@ -1,101 +1,98 @@
+      tasks.</span>
+</div>
 
-  <ng-template #tableSummary>
-    <div class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
-      <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }} </span>tasks.</span></div>
-  </ng-template>
+<ng-template #tableSummary>
+  <div class="pagination-top" attr.aria-label="{{ tasksTotal }} results have been found" role="status">
+    <span class="text-16" id="search-result-summary__text">Displaying <span class="govuk-!-font-weight-bold">{{ tasksTotal || 0 }} </span>tasks.</span></div>
+</ng-template>
 
-  <!-- Reset Sorting Button -->
-  <div *ngIf="showResetSortButton" class="reset-sort-button"><button type="button" class="govuk-button hmcts-button--secondary" (click)="onResetSorting()">Reset sorting</button></div>
+<!-- Reset Sorting Button -->
+<div *ngIf="showResetSortButton" class="reset-sort-button"><button type="button" class="govuk-button hmcts-button--secondary" (click)="onResetSorting()">Reset sorting</button></div>
 
-  <!-- Using the Angular Component Development Kit Table Module, and not Angular Material -->
-  <table cdk-table [dataSource]="dataSource$" class="govuk-table" [multiTemplateDataRows]="true">
+<!-- Using the Angular Component Development Kit Table Module, and not Angular Material -->
+<table cdk-table [dataSource]="dataSource$" class="govuk-table" [multiTemplateDataRows]="true">
 
-    <!-- Column Headers and Cells -->
-  <ng-container *ngIf="tasksTotal">
-    <ng-container *ngFor="let field of fields | paginate: { itemsPerPage: pagination?.page_size, currentPage: pagination?.page_number, totalItems: tasksTotal }" [cdkColumnDef]="field.name">
-      <th class="govuk-table__header" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
-        <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
-                (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
-      </th>
-      <td class="govuk-table__cell border-none" cdk-cell *cdkCellDef="let rowData">
-        <exui-task-field [task]="rowData" [config]="field"></exui-task-field>
-      </td>
-    </ng-container>
+  <!-- Column Headers and Cells -->
+<ng-container *ngIf="tasksTotal">
+  <ng-container *ngFor="let field of fields | paginate: { itemsPerPage: pagination?.page_size, currentPage: pagination?.page_number, totalItems: tasksTotal }" [cdkColumnDef]="field.name">
+    <th class="govuk-table__header" [id]="'header_' + field.sortName" [attr.aria-sort]="getColumnSortedSetting(field.sortName)" cdk-header-cell *cdkHeaderCellDef>
+      <button class="column-sort" *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.sortName" data-index="0"
+              (click)="onSortHandler(field.sortName)" >{{field.columnLabel}}</button>
```

Comment:
_Nitpick:_
`...(field.sortName)" >`

That extraneous space before the closing `>`.

Reply:
Not seeing this?